### ISSUE TITLE
[Bug]: Not all gmdl hashes are different.

### ISSUE THREAD

**Issue by Tydium** [Kas-tle/java2bedrock.sh#19821]
### Conversion Issue Link

None (self hosted)

### Expected Output

Below in the actual output I had highlighted the config.json entries where the gmdl hashes are the exact same. They both should be entirely different. 

### Actual Output

![image](https://github.com/Kas-tle/java2bedrock.sh/assets/67938521/33ae063f-7b25-44fa-bbcb-9168bce3e714)

Notice how both models are completely different yet they both still generate the same hash.

**Comment by itslilscorp** [comment]
Just came across the same issue myself. Notepad++ seems to be counting over 1k items with the same hash. 
![Screenshot 2024-03-27 114214](https://github.com/Kas-tle/java2bedrock.sh/assets/84101121/fb65c828-62e0-4b63-bd5e-acec1ea63a47)

Here's my conversion link:
https://github.com/Kas-tle/java2bedrock.sh/issues/23013

**Comment by Tydium** [comment]
Yeah I had noticed this while modifying the script. Im not entirely sure how I would fix it so Id rather leave it to the experts.
